### PR TITLE
Revives Geeves' PR for Uncloaked Action GUI elements.

### PR DESCRIPTION
**Pull request by LynxSolstice**
# About the pull request

This PR is a direct revival of https://github.com/cmss13-devs/cmss13/pull/1842 , it fixes the issue https://github.com/cmss13-devs/cmss13/issues/1665 .

# Explain why it's good for the game

Quoted from #1842, "[...] it was a bit silly that you could track things down not by seeing them, but by seeing the game UI floating above their heads. This should make it a little more engaging to track down the bad guy while giving them a chance to Metal Gear their way out of trouble."

# Testing Photographs and Procedure

<details>
<summary>It runs and cloaked players no longer have interaction circles above their heads. </summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

# Changelog

:cl: LynxSolstice, Geeves
balance: interaction circles that appear of above your head while cloaked are now harder to see (does not include circles above interacted with objects (ie. nests, C4s, breaching charges, xenos eating people, whatever predators have). That is to say, interaction circles inherit the alpha of cloaked users.
/:cl:

### CODE REVIEW

**Comment by realforest2001** [comment]
> Doesn't the wheel inherit carrier's alpha?

this is a better solution than disabling it entirely, and is better from a balance perspective too

_Originally posted by @stanalbatross in https://github.com/cmss13-devs/cmss13/issues/1842#issuecomment-1337422294_

**Comment by realforest2001** [comment]
Can you make above changes please, as they were requested on the original PR

**Comment by LynxSolstice** [comment]
> Can you make above changes please, as they were requested on the original PR
yes I can try and implement that

**Comment by LynxSolstice** [comment]
> Can you make above changes please, as they were requested on the original PR

Done!
https://imgur.com/a/kCnDS0c

File: code/__HELPERS/unsorted.dm [hunk 1]
```diff
@@ -1105,46 +1108,54 @@ var/global/image/action_purple_power_up
...
-		if(!istype(L) || has_target && !istype(target)) // Checks if L exists and is not dead and if the target exists and is not destroyed
+		if(show_busy_icon)
+			if(has_busy_icon && user.alpha < 255)
+				busy_user.overlays += busy_icon
+				busy_icon.alpha = user.alpha
+				has_busy_icon = TRUE
+			else if(!has_busy_icon && user.alpha == 255)
+				busy_user.overlays += busy_icon
+				has_busy_icon = TRUE
+		if(!istype(busy_user) || has_target && !istype(target)) // Checks if busy_user exists and is not dead and if the target exists and is not destroyed
```
Comment (stanalbatross) [inline code/__HELPERS/unsorted.dm:1119]: sanity check should come before the code block using busy_user I think

**Review by stanalbatross — CHANGES_REQUESTED** [review]

File: code/__HELPERS/unsorted.dm (hunk 1 above)
Comment (LynxSolstice) [reply code/__HELPERS/unsorted.dm:1119]: What's a sanity check? Is that 
> if(!istype(busy_user) || has_target && !istype(target)) // Checks if busy_user exists and is not dead and if the target exists and is not destroyed

Comment (LynxSolstice) [reply code/__HELPERS/unsorted.dm:1119]: Nevermind I see

**Review by stanalbatross — APPROVED** [review]

**Pull request merged**